frontend/src/main/pages/AdminStudentsCreatePage.js

```diff
@@ -0,0 +1,12 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+export default function AdminStudentsCreatePage() {
+  // Stryker disable all : placeholder for future implementation
```

Comment:
Are these stryker disables necessary? I think your tests should cover mutations here

Reply:
just tested it locally, they should be safe to remove

Reply:
removed them and verified that mutation testing still passes